app/src/main/java/org/jellyfin/androidtv/model/itemtypes/ChapterInfo.kt

```diff
@@ -0,0 +1,10 @@
+package org.jellyfin.androidtv.model.itemtypes
+
+import org.jellyfin.apiclient.model.dto.ChapterInfoDto
+import org.jellyfin.apiclient.model.entities.ImageType
+
+class ChapterInfo(originalChapterInfoDto: ChapterInfoDto, val baseItem: BaseItem, index: Int) {
```

Comment:
Can't you get the image from the data inside ChapterInfoDto?

Reply:
Sadly not. A ChapterInfoDto doesn't contain the ID of the item it belongs to, and the index is also required by the server, inquiring with tag is not enough :/

Reply:
Oh that's unfortunate. Can't do anything about it though :(

Reply:
Yeh, bummed me out as well that this old API is sabotaging us here. At least it's clean on the lifted ones